Content.Server/DeltaV/Biscuit/BiscuitComponent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Content.Shared.DeltaV.Biscuit;
+
+namespace Content.Server.DeltaV.Biscuit;
+
+/// <summary>
+/// This is used for...
+/// </summary>
+[RegisterComponent]
+public sealed partial class BiscuitComponent : SharedBiscuitComponent
+{
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("cracked")]
```

Comment:
("cracked") not required + vvrw is implied for datafields whenever you update engine, so you can remove that too after done with all the testing of it

Reply:
They get VVR by default, RW needs to be explicitly defined iirc.